src/components/SuccessConfModal.tsx

```diff
@@ -75,36 +83,54 @@ export const SuccessConfModal: FC<SuccessConfModalTypes> = props => {
           variant="body2"
         >
           <p>
-            Thank you for adding your case to the world map. You're helping us
-            understand COVID-19 so that we can fight it together. Please share
-            this with friends who are unwell. Send us your{' '}
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://docs.google.com/forms/d/e/1FAIpQLSfkQJMihQUlA6scYvjr1A1OZiXGRRxQLkD1YIiklGDq5YTclQ/viewform?usp=sf_link"
-            >
-              thoughts and feedback
-            </a>
-            . Sign up for email updates on the project.
+            Thank you for contributing! So far, {submissions && submissions.length} users like you have
+            added themselves to the world map.
           </p>
           <p>
-            To get support and learn more about COVID-19, please visit the{' '}
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019"
-            >
-              World Health Organization’s COVID-19 page
-            </a>
-            .
+            <ShareButtons/>
           </p>
           <p>
-            <ShareButtons />
-          </p>
-          <p>
-            COMING SOON: Get updates on current infections in your area and how
-            they are changing.
+            <DialogContentText
+              id="alert-dialog-slide-description"
+              variant="body2"
+            >
+              What you can do next:
+            </DialogContentText>
```

Comment:
I didn't know `<DialogContentText>` was a thing (did I use it before? haha). Good to know! Couple things:

1. I don't think `DialogContentText` should be nested into another `DialogContentText`. The MUI Dialog stuff is pretty flat in that sense (and really MUI in general, except for `<Grid>`). Honestly we only need the `simpler-font` class so 
1. I think what you want for "what you can do next" is to wrap it in `<Typography variant="h4">`, and take it out of the `<p>` (maybe take `<ol>` out of the `<p>` as well).
1. There's already an element with `id=alert-dialog-slide-description`, so a second instance is not valid.

Re: nesting, it's also causing some warnings like:

> `Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.`

I may have been guilty of this but since we're already in here we should fix it...